src/f64.rs

```diff
@@ -141,7 +136,7 @@ impl Float for F64 {
     }
 
     fn to_u32(&self, rnd: RoundingMode, exact: bool) -> u32 {
-        let ret = unsafe { softfloat_sys::f64_to_ui32(self.0, rnd.to_softfloat(), exact) };
+        let ret = unsafe { softfloat_sys::f64_to_ui64(self.0, rnd.to_softfloat(), exact) };
```

Comment:
Why?

Reply:
bold hack around c99 types mismatch. fixed to correct type